test/aspect/custom_config/ignore_include_patterns.json

```diff
@@ -2,6 +2,6 @@
     "ignore_include_patterns": [
         "foo/.*",
         "bar/.*",
-        "./ignored_sub_path/.*"
+        "\\w+/ignored_sub_path/.*"
     ]
```

Comment:
Should we add also an acceptance case here for ingore_include_patterns to mismatch?

Reply:
Good proposal. I will do so.